be/src/exec/pipeline/fragment_context.h

```diff
@@ -26,6 +26,7 @@ class FragmentContext {
 public:
     FragmentContext() : _cancel_flag(false) {}
     ~FragmentContext() {
+        _drivers.clear();
```

Comment:
Do we must explicitly call clear?

Reply:
ensure that drivers is destructed before pipeline::close method is invoked to prevent driver hold references to objects owned by pipelines.